docs/checkers/typescript.md

```diff
@@ -23,3 +23,4 @@ Advanced object configuration table of `options.typescript`.
 | root         | `string`  | [Vite config](https://vitejs.dev/config/#root) `root` | Root path to find tsconfig file                                                                                                                                                                                                 |
 | tsconfigPath | `string`  | `"tsconfig.json"`                                     | Relative tsconfig path to `root`                                                                                                                                                                                                |
 | buildMode    | `boolean` | `false`                                               | Add [`--build`](https://www.typescriptlang.org/docs/handbook/project-references.html) to `tsc` flag, note that `noEmit` does NOT work if `buildMode` is `true` ([#36917](https://github.com/microsoft/TypeScript/issues/36917)) |
+dev.debounce | `number` | `undefined` | Avoid multiple checks in a short time when your editor set lint on save action |  
```

Comment:
I'm assuming that `number` represents milliseconds. It should be explicitly listed so nobody has to guess.